addons/crossword/test/VisibilityTests.js

```diff
@@ -0,0 +1,83 @@
+TestCase('[Crossword] Visibility tests', {
+    setUp: function () {
+        this.presenter = Addoncrossword_create();
+
+        this.model = {
+            ID: "crossword1",
+        }
+
+        this.stubs = {
+            initializeLogic: sinon.stub(),
+            setVisibility: sinon.stub(),
+        };
+
+        this.presenter.initializeLogic = this.stubs.initializeLogic;
```

Comment:
brak czyszczenia mockowanych metod

Reply:
poprawione. Dobrze że to u mnie złapałeś :)